### PR TITLE
core/manager: do not re-init Manager.lookup_paths when manager_reload(), minor assorted cleanups

### PR DESCRIPTION
**Pull request by YHNdnzj**

### CODE REVIEW

File: src/core/manager.c [hunk 1]
```diff
@@ -3713,9 +3713,7 @@ int manager_reload(Manager *m) {
         m->uid_refs = hashmap_free(m->uid_refs);
         m->gid_refs = hashmap_free(m->gid_refs);
 
-        r = lookup_paths_init_or_warn(&m->lookup_paths, m->runtime_scope, 0, NULL);
-        if (r < 0)
-                return r;
+        assert_se(lookup_paths_init_or_warn(&m->lookup_paths, m->runtime_scope, 0, NULL) >= 0);
```
Comment (bluca) [inline src/core/manager.c]: An assert here makes me a bit nervous, as this reads from the env vars... isn't there anything else that can be done?

Comment (YHNdnzj) [reply src/core/manager.c]: Hmm, the env of manager shouldn't change for the whole runtime, right? If the initial setup in manager_startup() succeeded but fails here, I think the system is totally borked...

Comment (yuwata) [reply src/core/manager.c]: How about to call this earlier on a temporal object, then take it here?
```c
_cleanup_(lookup_paths_done) LookupPaths lookup_paths = {};
r = lookup_paths_init_or_warn(&m->lookup_paths, m->runtime_scope, 0, NULL);
if (r < 0)
        return r;

/* 💀 This is the point of no return, from here on there is no way back. 💀 */
reloading = NULL;

...

m->lookup_paths = TAKE_STRUCT(lookup_paths);
```

`assert()` or `assert_se()` should be used for checking programming error. And the failure here should not.

**Review by yuwata — CHANGES_REQUESTED** [review]

File: src/core/manager.c [hunk 2]
```diff
@@ -101,8 +101,8 @@
 #include "virt.h"
 #include "watchdog.h"
 
-#define NOTIFY_RCVBUF_SIZE (8*1024*1024)
-#define CGROUPS_AGENT_RCVBUF_SIZE (8*1024*1024)
+/* Make sure clients notifying us don't block */
+#define MANAGER_SOCKET_RCVBUF_SIZE (8*U64_KB)
```
Comment (yuwata) [inline src/core/manager.c]: U64_KB -> U64_MB ??

**Review by yuwata — CHANGES_REQUESTED** [review]

File: src/core/manager.c (hunk 2 above)
Comment (YHNdnzj) [reply src/core/manager.c]: ugh

**Review by yuwata — APPROVED** [review]
LGTM.

**Comment by poettering** [comment]
lgtm

**Pull request merged**